openlibrary/core/models.py

```diff
@@ -644,45 +644,48 @@ def resolve_redirects_bulk(
         cls,
         batch_size=1000,
         start_offset=0,
-        limit=1000,
         grace_period_days=7,
         cutoff_date=datetime.datetime(year=2017, month=1, day=1),
         test=True,
     ):
         """
         batch_size - how many records to fetch per batch
         start_offset - what offset to start from
-        limit - total number of records to process from start_offset
         grace_period_days - ignore redirects created within period of days
         cutoff_date - ignore redirects created before this date
         test - don't resolve stale redirects, just identify them
         """
+        batch = 0
         pos = start_offset
-        max_limit = start_offset + limit
         grace_date = datetime.datetime.today() - datetime.timedelta(
             days=grace_period_days
         )
-        batch_offsets = enumerate(range(start_offset, max_limit, batch_size))
-        for batch, offset in batch_offsets:
+
+        go = True
+        while go:
```

Comment:
You don't need this variable since you're `break`-ing.
```suggestion
        while True:
```

Reply:
We do because the break occurs 2 layers deep -- there's a for loop within the while loop
https://github.com/internetarchive/openlibrary/blob/551079e0caa7262480b44efa4e30692ebfb36656/openlibrary/core/models.py#L689

Reply:
Ah didn't see the second loop 👍